### ISSUE TITLE
WARN about System.out/err.print

### ISSUE THREAD

**Issue by SlimeDog** [Jumper251/AdvancedReplay#39]
Paper 1.17.1-151
AdvancedReplay 1.6.6

Paper today changed their logger. On server start-up, the log shows (twice)
```
[08:11:09] [Server thread/WARN]: Nag author(s): '[Jumper251]' of 'AdvancedReplay' about their usage of System.out/err.print. Please use your plugin's logger instead (JavaPlugin#getLogger).
```

**Comment by Jumper251** [comment]
Fixed in the latest update.

**Comment by SlimeDog** [comment]
Confirmed fixed in 1.7. Thanks.
